src/Config/DatabasePrimitives/DatabaseObject.cs

```diff
@@ -216,6 +232,7 @@ public class RelationshipMetadata
     /// <summary>
     /// Dictionary of target entity name to ForeignKeyDefinition.
     /// </summary>
+    [JsonInclude]
```

Comment:
do you not need to add a constructor for this class as well?

Reply:
nope. Without constructor it works